drivers/start_date_dq/app/models/start_date_dq/report.rb

```diff
@@ -0,0 +1,78 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module StartDateDq
+  class Report
+    include ActionView::Helpers::NumberHelper
+    include ArelHelper
+    include Filter::FilterScopes
+
+    attr_reader :filter
+
+    def initialize(user_id, filter = nil)
+      @filter = filter || default_filter(user_id)
+    end
+
+    def default_filter(user_id)
+      Filters::FilterBase.new(user_id: user_id, start: '2018-01-01')
+    end
+
+    def self.viewable_by(user)
+      GrdaWarehouse::WarehouseReports::ReportDefinition.where(url: url).
+        viewable_by(user).exists?
+    end
+
+    def self.url
+      'start_date_dq/warehouse_reports/reports'
+    end
+
+    def title
+      'Approximate Start Date Data Quality'
+    end
+
+    def column_names
+      ['# Days Difference',
+       'DateToStreetESSH',
+       'Entry Date',
+       'Personal ID',
+       'Project',
+       'Project Type']
+    end
+
+    def column_values(row)
+      date_to_street = row.enrollment.DateToStreetESSH
+      entry_date = row.enrollment.EntryDate
+      [
+        (entry_date - date_to_street).to_i,
```

Comment:
One more question for @eanders: In my test data, there are some enrollments where the Entry Date is _before_ the DateToStreetESSH. Those show up on the last page as negative numbers. Is that a scenario that may occur in prod? If so, maybe we should be taking the absolute value here.

Reply:
Showing the negative number is the right thing to do.  That may be one of the data quality issues we encounter.  Won't know until we run in production though.